docs/src/content/docs/commands/JSON.FORGET.md

```diff
@@ -3,107 +3,92 @@ title: JSON.FORGET
 description: Documentation for the DiceDB command JSON.FORGET
 ---
 
-The `JSON.FORGET` command is part of the DiceDBJSON module, which allows you to manipulate JSON data stored in DiceDB. This command is used to delete a specified path from a JSON document stored at a given key. If the path leads to an array element, the element is removed, and the array is reindexed.
+The `JSON.FORGET` command in DiceDBJSON is used to delete a specified path from a JSON document stored at a given key. If the path leads to an array element, the element is removed, and the array is reindexed. This is useful for modifying and updating portions of a JSON document.
```

Comment:
Should this be `The JSON.FORGET command in DiceDB is used to delete a specified path from a JSON document stored at a given key.`?